test/src/test/resources/fixtures/introspection/if-there-are-a-mix-of-system-fields-and-normal-fields-it-works-on-nextgen.yml

```diff
@@ -0,0 +1,85 @@
+name: if there are a mix of system fields and normal fields it works on nextgen
+enabled:
+  current: false
```

Comment:
does it not work on current gen?

Reply:
yeah, you cannot run a mixture of system and normal fiedls on the current gen https://github.com/atlassian-labs/nadel/blob/d790fcdc9cdd11b8cc6ff7446023a42c56aee218/api/src/main/java/graphql/nadel/introspection/DefaultIntrospectionRunner.java#L44